hack/jenkins/test-dockerized.sh

```diff
@@ -19,33 +19,14 @@ set -o nounset
 set -o pipefail
 set -o xtrace
 
-# Runs test-cmd and test-integration,
-# producing JUnit-style XML test
-# reports in ${WORKSPACE}/artifacts. This script is intended to be run from
-# kubekins-test container with a kubernetes repo mapped in. See
-# k8s.io/test-infra/scenarios/kubernetes_verify.py
+# Runs test-cmd and test-integration, intended to be run in prow.k8s.io
+set -x;
 
-export PATH=${GOPATH}/bin:${PWD}/third_party/etcd:/usr/local/go/bin:${PATH}
+# TODO: make test-integration should handle this automatically
+source ./hack/install-etcd.sh
 
-# Install tools we need
-hack_tools_gotoolchain="${GOTOOLCHAIN:-}"
-if [ -n "${KUBE_HACK_TOOLS_GOTOOLCHAIN:-}" ]; then
-  hack_tools_gotoolchain="${KUBE_HACK_TOOLS_GOTOOLCHAIN}";
-fi
-GOTOOLCHAIN="${hack_tools_gotoolchain}" go -C "./hack/tools" install gotest.tools/gotestsum
```

Comment:
this should be useless, or if not, we should fix that, `make test` automatically handles gotestsum, I think we still have to add that to these (PR is WIP ....)

Reply:
We ultimately run gotestsum via `make test` so we don't need this.